src/plasmapy/diagnostics/thomson.py

```diff
@@ -682,7 +682,7 @@ def _params_to_array(
 
 def _spectral_density_model(wavelengths, settings=None, **params):
     """
-    lmfit Model function for fitting Thomson spectra.
+    `lmfit` model function for fitting Thomson spectra.
```

Comment:
...since one of the pre-commit hooks now capitalizes the beginning of docstrings.